src/plugins/discover/public/application/components/create_discover_grid_directive.tsx

```diff
@@ -25,6 +25,7 @@ export function DiscoverGridEmbeddable(props: DiscoverGridProps) {
         setExpandedDoc={setExpandedDoc}
         expandedDoc={expandedDoc}
         services={getServices()}
+        data-shared-item
```

Comment:
if this is needed here, maybe it should be removed here:

https://github.com/elastic/kibana/blob/52dbe10a1c475129b4380fd5af0c8773b81a8e3a/src/plugins/discover/public/application/components/discover_grid/discover_grid.tsx#L328